Updating messaging to match all Teleport protocols (#10768)
gravitational/teleport: docs/pages/enterprise/sso.mdx

<TileSet>
  <Tile icon="bolt" title="Azure Active Directory (AD)" href="./sso/azuread.mdx">

>     Configure Azure Active Directory SSO for SSH, Kubernetes, Databases, Desktops and Web apps.

Reviewer: ```suggestion
    Configure Azure Active Directory SSO for SSH, Kubernetes, databases, desktops and web apps.
```

  </Tile>
  <Tile icon="bolt" title="Active Directory (ADFS)" href="./sso/adfs.mdx">

>     Configure Windows Active Directory SSO for SSH, Kubernetes, Databases, Desktops and Web apps.

Reviewer: ```suggestion
    Configure Windows Active Directory SSO for SSH, Kubernetes, databases, desktops and web apps.
```

  </Tile>
  <Tile icon="bolt" title="Google Workspace" href="./sso/google-workspace.mdx">

>     Configure Gsuite SSO for SSH, Kubernetes, Databases, Desktops and Web apps.

Reviewer: ```suggestion
    Configure Google Workspace SSO for SSH, Kubernetes, databases, desktops and web apps.
```

  </Tile>
  <Tile icon="bolt" title="GitLab" href="./sso/gitlab.mdx">

>     Configure Gitlab SSO for SSH, Kubernetes, Databases, Desktops and Web apps.

Reviewer: ```suggestion
    Configure GitLab SSO for SSH, Kubernetes, databases, desktops and web apps.
```

  </Tile>
  <Tile icon="bolt" title="OneLogin" href="./sso/one-login.mdx">

>     Configure OneLogin SSO for SSH, Kubernetes, Databases, Desktops and Web apps.

Reviewer: ```suggestion
    Configure OneLogin SSO for SSH, Kubernetes, databases, desktops and web apps.
```

  </Tile>
  <Tile icon="bolt" title="OIDC" href="./sso/oidc.mdx">

>     Configure OIDC SSO for SSH, Kubernetes, Databases, Desktops and Web apps.

Reviewer: ```suggestion
    Configure OIDC SSO for SSH, Kubernetes, databases, desktops and web apps.
```

  </Tile>
  <Tile icon="bolt" title="Okta" href="./sso/okta.mdx">

>     Configure Okta SSO for SSH, Kubernetes, Databases, Desktops and Web apps.

Reviewer: ```suggestion
    Configure Okta SSO for SSH, Kubernetes, databases, desktops and web apps.
```
